package/MDAnalysis/core/topology.py

```diff
@@ -473,6 +478,18 @@ def __init__(self, n_atoms=1, n_res=1, n_seg=1,
         for topologyattr in attrs:
             self.add_TopologyAttr(topologyattr)
 
+    def copy(self):
+        """Return a deepcopy of this Topology"""
+        new = self.__class__(1, 1, 1)
+        # copy the tt
+        new.tt = self.tt.copy()
+        # remove indices
+        for attr in self.attrs:
+            if isinstance(attr, (Atomindices, Resindices, Segindices)):
```

Comment:
why do we skip these attributes here?

Reply:
Also what class is this in? The diff on Github is hiding that.

Reply:
these attrs are added automatically on `Topology.__init__` so you don't need to copy them in